providers/os/connection/snapshot/blockdevices.go

```diff
@@ -225,6 +234,16 @@ func (device BlockDevice) GetPartitions(includeBoot bool, includeMounted bool) (
 	return partitions, nil
 }
 
+func mapLVM2Partitions(part BlockDevice, partitions *[]*PartitionInfo) {
```

Comment:
can we get this to reutnr the lv2m parts (children) and then append those to `partitions` in the caller of `mapLVM..`, would make for better readability

Reply:
can we also add tests for lvm2 parts?